nvim/init.lua

```diff
@@ -1,18 +1,11 @@
+vim.loader.enable()
```

Comment:
issue: Please move this back to where it was.
The reason I don't have it in `init.lua` is so that it is called *at the very beginning* of the generated `init.lua`.
I don't know if that has any performance impact right now, but it could in the future.

Reply:
Yes, as soon as we put any `require` calls in `nix`, it immediately makes difference.

It's safe to have this call in both places, by the way

Reply:
When it's only in nix, then there it's not called when wrapRc=false.

I explained it in the details of commit message, but it's hardly visible))

Reply:
Ah, good point. Let's keep it in init.lua only for now then.